action.yml

```diff
@@ -16,6 +16,8 @@ inputs:
 outputs:
   time: # id of output
     description: 'The time'
+  message-ts: # timestamp on the message that was posted when using bot token
```

Comment:
I have two things that I would like to discuss here:

1: always passing thread_ts?
What do you think about naming this `thread-ts` instead and making the slack-send to always return thread-ts? This means that if you run the step twice, you can easily post a message in the same thread. I cannot think of the use cases to prefer knowing message.ts instead of message.thread_ts.

The possible use case would be:

```yaml
steps:
  - uses: slackapi/slack-github-action@v1.x
    id: slack1
    with:
      channel-id: '#general'
      slack-message: "This is the first message"
  - uses: slackapi/slack-github-action@v1.x
    id: slack2
    with:
      channel-id: '#general'
      payload: |
        {
          "text": "The first reply",
          // Actually this is parent message's message.ts but the following steps use it as thread_ts
          "thread_ts": "${{ steps.slack1.outputs['thread-ts'] }}" 
        }
  - uses: slackapi/slack-github-action@v1.x
    id: slack3
    with:
      channel-id: '#general'
      payload: |
        {
          "text": "The second reply",
          // This should be the same ts with slack1's message ts
          "thread_ts": "${{ steps.slack2.outputs['thread-ts'] }}"
        }
```

2: thread-ts vs thread_ts?

I know using dash as delimiter is consistent with the existing input value keys. That being said, using underscopre can be a bit similar for users. In the YAML expression, the difference would be `steps.slack1.outputs['thread-ts']` vs `steps.slack1.outputs.thread_ts`.